src/main/java/th/ac/kmitl/soa/group9/taxInvoice/model/ExchangedDocumentTest.java

```diff
@@ -18,25 +15,4 @@
     private Timestamp creationDateTime = new Timestamp(System.currentTimeMillis());
     private IncludedNote includedNote = new IncludedNote();
```

Comment:
Where is the test ? If it is not test why named it test. Test it properly or remove it.